src/functions/postTechRecords.ts

```diff
@@ -35,14 +35,14 @@ const postTechRecords = (event: any) => {
     secondaryVrms
   };
 
-  return techRecordsService.insertTechRecord(techRecord, msUserDetails)
-    .then((data) => {
-      return new HTTPResponse(201, HTTPRESPONSE.TECHINICAL_RECORD_CREATED);
-    })
-    .catch((error: any) => {
-      console.log(error);
-      return new HTTPResponse(error.statusCode, error.body);
-    });
+  try {
+    console.log("inside post Tech Record function");
```

Comment:
Don't think this adds any value to the logs

Reply:
thanks for highlighting. Forgot to remove.